### ISSUE TITLE
Outputting .br files in production even when not using Brotli

### ISSUE THREAD

**Issue by JoshRobertson** [rails/webpacker#2399]
after the support was added for Brotli in 4.1.0, I noticed our prod build started emitting both `.gz` and `.br` without anything changing in our setup or configs except updating Webpacker.

Rolling back to 4.0.7 predictably solves the issue.

Is there some way to explicitly disable this output?

**Comment by njradford** [comment]
A config to disable .br output would be great.

**Comment by njradford** [comment]
Also wondering if there's a way to disable this

**Comment by arturopuente** [comment]
@JoshRobertson @njradford this works, but I'd love to see a more accesible config option exposed to disable brotli.

```javascript
// in production.js
const config = environment.toWebpackConfig();
// Disables brotli (.br) compression
config.plugins = config.plugins.filter(conf => {
  if (conf.options) {
    return conf.options.filename !== "[path].br[query]";
  } else {
    return conf;
  }
});
```

**Comment by dreyks** [comment]
you can do it before `toWebpackConfig`
```js
environment.plugins.delete('Compression Brotli')
```

but I'd really wanna see plugins section be more configurable through webpacker.yml maybe

**Comment by woodhull** [comment]
Yes, wish we could specify `compress: false` somewhere to skip both gzip and brotli compression, since we do this with our CDN.

**Comment by guillaumebriday** [comment]
You can remove the plugin in your configuration. But why would you do that? 🤔

**Comment by Cervenka** [comment]
If you want to skip both gzip and br compression:
```
config.plugins = config.plugins.filter(plugin => {
  return (plugin.constructor.name != 'CompressionPlugin')
})
```
